deebotozmo/cli.py

```diff
@@ -0,0 +1,439 @@
+        lock.release()
+
+    battery_listener = vacbot.bot.events.battery.subscribe(on_battery)
+    await lock.acquire()
+    vacbot.bot.events.battery.unsubscribe(battery_listener)
+
+    async def on_fan_event(event: FanSpeedEvent) -> None:
+        print("Fan Speed: " + str(event.speed))
+        lock.release()
+
+    fan_speed_listener = vacbot.bot.events.fan_speed.subscribe(on_fan_event)
+    await lock.acquire()
+    vacbot.bot.events.fan_speed.unsubscribe(fan_speed_listener)
+
+    async def on_water_level(event: WaterInfoEvent) -> None:
+        print("Water Level: " + str(event.amount))
+        lock.release()
+
+    water_level_listener = vacbot.bot.events.water_info.subscribe(on_water_level)
+    await lock.acquire()
+    vacbot.bot.events.water_info.unsubscribe(water_level_listener)
+
+    await vacbot.goodbye()
+
+
+@cli.command(help="Get stats")
+@coro
+async def stats() -> None:
+    """Click subcommand that returns bot stats."""
+    vacbot = DoLogin()
+    await vacbot.run()
+
+    lock = asyncio.Lock()
+    await lock.acquire()
+
+    async def on_stats_event(event: StatsEvent) -> None:
+        print("Stats Cid: " + str(event.clean_id))
+        print("Stats Area: " + str(event.area))
+        if isinstance(event.time, int):
+            print("Stats Time: " + str(int(event.time / 60)) + " minutes")
+        print("Stats Type: " + str(event.type))
+        lock.release()
+
+    listener = vacbot.bot.events.stats.subscribe(on_stats_event)
+    await lock.acquire()
+    vacbot.bot.events.stats.unsubscribe(listener)
+
+    await vacbot.goodbye()
+
+
+@cli.command(help="Get robot components life span")
+@coro
+async def components() -> None:
+    """Click subcommand that returns the robot's life span."""
+    vacbot = DoLogin()
+    await vacbot.run()
+
+    lock = asyncio.Lock()
+    await lock.acquire()
+
+    async def on_lifespan_event(event: dict) -> None:
+        for key, value in event.items():
+            print(f"{key}: {value}%")
+        lock.release()
+
+    listener = vacbot.bot.events.lifespan.subscribe(on_lifespan_event)
+    await lock.acquire()
+    vacbot.bot.events.lifespan.unsubscribe(listener)
+
+    await vacbot.goodbye()
+
+
+@cli.command(name="getrooms", help="Get saved rooms")
+@coro
+async def get_rooms() -> None:
+    """Click subcommand that returns saved room."""
+    vacbot = DoLogin()
+    await vacbot.run()
+
+    lock = asyncio.Lock()
+    await lock.acquire()
+
+    async def on_rooms(event: RoomsEvent) -> None:
+        for room in event.rooms:
+            print(str(room.id) + " " + str(room.subtype))
+        lock.release()
+
+    listener = vacbot.bot.map.events.rooms.subscribe(on_rooms)
+    await lock.acquire()
+    vacbot.bot.map.events.rooms.unsubscribe(listener)
+
+    await vacbot.goodbye()
+
+
+@cli.command(
+    name="exportlivemap",
+    help='Get robot map and save it [filepath ex: "/folder/livemap.png"',
+)
+@click.argument("filepath", type=click.STRING, required=True)
```

Comment:
As the image is a png, maybe add a check that the file ends with `png` or what you think?